brightlayer-ui/doc-it
# Valentine's Day Theme

Doing this to quench my thirst for themes during the first half of the year, as well as a design I've been wanting to do for 2 years.

https://user-images.githubusercontent.com/8997218/200684266-352c3a7a-b088-4fe1-aed3-67ea0c7dca42.mov

To test:
- change the `end` field on line 12 of `src/__configuration__/themes/valentine-s-day.ts` to: 
```js
end: new Date(1, 1, 15), 
```
- Observe the fancy change on docit landing page
